src/leap/mail/imap/mailbox.py

```diff
@@ -639,21 +533,41 @@ def _do_fetch(self, messages_asked, uid, d):
 
         sequence = False
         # sequence = True if uid == 0 else False
```

Comment:
This comment don't match the actual implementation where sequence is always equal to False.